thorlcr/activities/hashdistrib/thhashdistribslave.cpp

```diff
@@ -155,10 +155,11 @@ class CDistributorBase : implements IHashDistributor, implements IExceptionHandl
                     }
                     else
                     {
-                        prev.set(row);
-                        rows.enqueue(row.getClear());
+                        rows.enqueue(prev.getClear());
+                        prev.setown(row.getClear());
```

Comment:
That 1st line (rows.enqueue(prev.getClear());) was not supposed to be in this commit.